examples/controls/webpack.config.js

```diff
@@ -0,0 +1,65 @@
+// NOTE: This is a Webpack 2 configuration file for react-map-gl
+const {resolve} = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  // Example entry point
+  entry: {
+    app: resolve('./app.js')
+  },
+
+  // Silence excessive webpack dev server warnings
+  devServer: {
+    stats: {
+      warnings: false
+    }
+  },
+
+  devtool: 'source-maps',
+
+  resolve: {
+    alias: {
+      // Work against the latest base library in this repo
+      'react-map-gl': resolve('../..'),
+      // Ensure only one copy of react
+      react: resolve('./node_modules/react'),
+      immutable: resolve('./node_modules/immutable'),
+      // Per mapbox-gl-js README for non-browserify bundlers
+      'mapbox-gl$': resolve('./node_modules/mapbox-gl/dist/mapbox-gl.js')
+    }
+  },
+
+  module: {
+    rules: [
+      {
+        enforce: 'pre',
```

Comment:
Not sure, all copied from `webpack-exhibit`. We should clean up that project as well.